docs/ADR-5-cardano-testnet-node-configuration-file.md

```diff
@@ -0,0 +1,42 @@
+# Status
+
+📜 Proposed 2024-09-10
+
+# Context
+
+There is no way to check a node configuration file for basic sanity issues.
```

Comment:
Can you describe the risks and who is affected by not checking the configuration file?
I see that you described it in the *Consequences*, but I think it belongs more to the *Context*.

Reply:
Good point :+1: 

Adding more details in the _Context_ section:

```diff
 # Context
 
-There is no way to check a node configuration file for basic sanity issues.
+There is no way to check a node configuration file for basic sanity issues. This is a problem for people running testnets, because they will only discover _after having started their testnet_ that something is wrong (because the node will refuse to start). And starting a testnet is a costly operation, meaning the failures will only happen after tenth of seconds; causing the feedback loop to be slow.
```